examples/stress_tests/many_components.rs

```diff
@@ -0,0 +1,194 @@
+//! Stress test for large ECS worlds.
+//!
+//! Running this example:
+//!
+//! ```
+//! cargo run --profile stress-test --example many_components <num_entities> <num_components> <num_systems>
```

Comment:
```suggestion
//! cargo run --profile stress-test --example many_components [<num_entities>] [<num_components>] [<num_systems>]
```

Maybe also otherwise highlight that there are quite sane defaults, so values only need to be specified if testing something custom.